alea/models/blueice_extended_model.py

```diff
@@ -138,11 +139,31 @@ def get_source_name_list(self, likelihood_name: str) -> list:
         ll_index = self.likelihood_names.index(likelihood_name)
         return self.likelihood_list[ll_index].source_name_list
 
+    def get_all_source_name(self) -> set:
+        """Return a set of possible source names from all likelihood terms.
+
+        Args:
+            likelihood_name (str): Name of the likelihood.
+
+        Returns:
+            set: set of source names.
+
+        """
+        source_name = set(
+            itertools.chain.from_iterable([ll.source_name_list for ll in self.likelihood_list[:-1]])
+        )
+        return source_name
+
```

Comment:
This is not used anywhere. I would remove it

Reply:
also, it doesn't take `likelihood_name` and I think if at all it should be `get_all_source_names`.

Reply:
Suspended by https://github.com/XENONnT/alea/pull/111